framework/material/Form.kt

```diff
@@ -137,11 +136,11 @@ object Form {
         val color: @Composable () -> Color = { Theme.studio.border }
     )
 
-    data class IconArg(val code: Icon.Code, val color: @Composable () -> Color = { Theme.studio.icon })
+    data class IconArg(val code: Icon, val color: @Composable () -> Color = { Theme.studio.icon })
```

Comment:
```suggestion
    data class IconArg(val icon: Icon, val color: @Composable () -> Color = { Theme.studio.icon })
```
There may be other instances of this in the codebase.

Reply:
Done.